.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+<!-- Credit (as a starting template): https://github.com/visgl/deck.gl/blob/master/.github/pull_request_template.md -->
+<!-- For feature, feature enhancement or bug fix, create an issue first and finish To Do List there -->
+<!-- Anything doesn't work as expected is a bug, including code, doc and test -->
+Fixes #
+<!-- For other PRs without open issue -->
+#### Background
+<!-- For all the PRs -->
+#### Change List
+-
+#### Checklist
+ - [ ] Ensure PR works with all demos on the vitessce.io homepage
+ - [ ] Open (draft) PR's into [`vitessce-python`](https://github.com/vitessce/vitessce-python) and [`vitessce-r`](https://github.com/vitessce/vitessce-r) if the schema has changed
```

Comment:
```suggestion
 - [ ] Open (draft) PR's into [`vitessce-python`](https://github.com/vitessce/vitessce-python) and [`vitessce-r`](https://github.com/vitessce/vitessce-r) if this is a release PR
```

I like this!

Since we currently deploy to NPM _after_ merging the release PR into master here, should we commit to keeping the Python and R packages at most one JS version behind?

Reply:
I think this is fine.  My main goal was to avoid having to periodically go through past commits looking for changes/additions so doing this at PR time also seems fine.

Reply:
This will also allow us to do iteration within releases without having to make a bunch of PR's into the packages so this is good.